cmd/bosun/conf/system_elastic_all.go

```diff
@@ -20,7 +21,7 @@ func parseESConfig(sc *SystemConf) expr.ElasticHosts {
 		case expr.ESV6:
 			cfg = parseESConfig6(value)
 		default:
-			panic(fmt.Errorf("conf: invalid elastic version: %s (supported versions are: v2, v5, v6)", value.Version))
+			slog.Fatal(fmt.Errorf("conf: invalid elastic version: %s (supported versions are: v2, v5, v6)", value.Version))
```

Comment:
For this and the annotate error we should include:
- If it is for annotate or elastic, and also in the case of elastic which prefix it is (i.e. "default" or another named instance).
- I think since this is a breaking change, we should add a case for an empty version and return an error message saying that is a required parameter (including the info from the above bullet point)